spec/dummy/app/views/components/modal.slim

```diff
@@ -0,0 +1,6 @@
+button#show Show
+
+= ui_component 'modal', \
```

Comment:
Not sure. This would add a second place where you have to add something to make it work. I was also thinking about taking the url, the remote content would come from, from the `href` attribute of the triggering button/link, UJS style, but that's probably not a concern of ours, is it? It also, again, adds one more place where you have to put stuff when using it. What do you think?